pkg/osm/annotations.go

```diff
@@ -0,0 +1,16 @@
+package osm
+
+// SupportedAnnotations maintains a mapping of OSM version to supported annotations.
+var SupportedAnnotations = map[ControllerVersion][]Annotation{
+	"v0.5": {},
+	"v0.6": {},
+	"v0.7": {},
+	"v0.8": {},
+	"v0.9": {
+		// Source: https://github.com/openservicemesh/osm/releases/tag/v0.9.0
+		"openservicemesh.io/outbound-port-exclusion-list"},
+	"v0.9.1": {
```

Comment:
I'd recommend we keep this at `v0.9` - major + minor and skip the patch.  By convention we'd expect no significant changes (no new annotations etc. added) between patch versions.

Reply:
@draychev quick question - if an annotation is introduced in v0.9.1 for example, should the support matrix in this table be `v0.9` or `v0.10`?

Reply:
If an annotation is introduced in v0.9.1 --- then our [release-v0.9](https://github.com/openservicemesh/osm/tree/release-v0.9) branch will have it.